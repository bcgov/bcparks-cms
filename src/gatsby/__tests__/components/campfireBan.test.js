import React from "react"
import renderer from "react-test-renderer"
import CampfireBan from "../../src/components/campfireBan"

// Sample test adapted from https://www.gatsbyjs.com/docs/how-to/testing/unit-testing/
describe("CampfireBan", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<CampfireBan />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})