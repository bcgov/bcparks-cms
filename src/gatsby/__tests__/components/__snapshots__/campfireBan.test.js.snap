// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CampfireBan renders correctly 1`] = `
Array [
  <img
    alt="Campfire ban in effect"
    src="test-file-stub"
  />,
  <p>
    Campfire ban in effect.
  </p>,
]
`;
